prime-router/src/main/kotlin/cli/OktaCommands.kt

```diff
@@ -62,7 +62,7 @@ class LoginCommand : OktaCommand(
         .default("local", "local")
 
     override fun run() {
-        val oktaApp = Environment.get().oktaApp ?: abort("No need to login in this environment")
+        val oktaApp = Environment.get(env).oktaApp ?: abort("No need to login in this environment")
```

Comment:
env is the `option` type defined in the Command. We need to hook it up to the environment that is used.